<launch>
  <!-- Launch trajectory generation node -->
  
  <node name="trajectory_generation_node" pkg="trajectory_generation" type="toan_marker" output="screen" />

  <node name="controller_node" pkg="controller_pkg" type="controller_node"/>

  <!-- Controller gains -->
  <param name="kx" type="double" value="0.5" />
  <param name="kv" type="double" value="0.3" />
  <param name="kr" type="double" value="0.8" />
  <param name="komega" type="double" value="0.2" />
  <!-- TODO: Copy your controller gains from lab 3 to here! -->
</launch>
